apps/staking/components/Maintenance.tsx

```diff
@@ -0,0 +1,43 @@
+import { ButtonDataTestId } from '@/testing/data-test-ids';
+import { Button } from '@session/ui/ui/button';
+import Link from 'next/link';
+import { getTranslations } from 'next-intl/server';
+import { TriangleAlertIcon } from '@session/ui/icons/TriangleAlertIcon';
+import { cn } from '@session/ui/lib/utils';
+import { SOCIALS } from '@/lib/constants';
+import { Social } from '@session/ui/components/SocialLinkList';
+
+export default async function Maintenance() {
+  const dict = await getTranslations('maintenance');
+  return (
+    <div className="-mt-header-displacement flex h-screen w-full items-center justify-center p-32 align-middle">
+      <div className="flex flex-col items-center justify-center text-center align-middle md:flex-row md:gap-10 md:text-left">
+        <TriangleAlertIcon className={cn('stroke-warning h-60 w-60 stroke-2')} />
+        <div className="flex h-full flex-col justify-between gap-3 md:gap-4">
+          <p className="text-md text-warning max-w-sm md:text-xl">{dict('title')}</p>
+          <p className="text-md max-w-sm md:text-xl">{dict('description')}</p>
+          <div className="flex flex-col justify-center gap-4 md:flex-row md:justify-start">
+            <Link href={SOCIALS[Social.Discord].link} prefetch={false}>
```

Comment:
Treat these as external links and open in a new tab.

Reply:
But the current tab is useless, we want to direct people away from the maintenance page and towards info about it.

Reply:
If they came through to the site from a link such as a referral code it's not useless as the url will be maintained in the tab.

Reply:
But you cant do anything, the site is down. you'd need to come back later anyway. the reason you might choose to make a link open in an external tab is if it takes the user off the primary path, like away from an app or the main site content, but if the site is down, the main path is "go away from here and look at this link, that is where updates will be"

Reply:
That being said I don't think it matters, i just think its annoying for the site to open the link in a new tab in this case.

Reply:
Cool then resolving.